service/config.go

```diff
@@ -20,8 +20,9 @@ type CommandVariables struct {
 
 // Config represents the configuration.
 type Config struct {
-	Version  string
-	Commands map[string]map[string]string `yaml:"commands"`
+	Version   string
+	Commands  map[string]map[string]string `yaml:"commands"`
+	ImageURIs []string                     `yaml:"image_uris"`
```

Comment:
Wondering if `image_uris` should be a top level config item? This effectively scopes it globally but wondering if it should be (or optionally) by exec/pre or by command? This kind of gets into the "does the config structure need to be more thoroughly/flexibly designed?" comment.

Reply:
Agreed, this leads to that config structure question. Feels a bit awkward as a top level item, but while I was pondering through options, I didn't want to land in a config where (for instance) the CDK image was allowed for diff, but not sync and you get a failure later in the pipeline sequence than expected. Very much open to suggestions on improvements to the "where" for this.